libs/blocks/global-navigation/global-navigation.js

```diff
@@ -869,6 +869,9 @@ class Gnav {
         linkElem.setAttribute('daa-ll', getAnalyticsValue(linkElem.textContent, index + 1));
         if (itemHasActiveLink) {
           linkElem.removeAttribute('href');
+          linkElem.setAttribute('role', 'link');
+          linkElem.setAttribute('aria-disabled', 'true');
+          linkElem.setAttribute('aria-current', 'page');
```

Comment:
Are there any unit tests related to this use-case that we might want to expand?

Reply:
There is one, thanks for pointing it out 👍. I expanded to also test that the new attributes are present.